### PR TITLE
[zh] Sync #14858 1.19.9, 1.20.5, 1.21.1 and Security Advisory 2024-002 into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
## Description

Sync #14858 1.19.9, 1.20.5, 1.21.1 and Security Advisory 2024-002 into Chinese

## Reviewers

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure
- [x] Localization/Translation

### CODE REVIEW

File: content/zh/news/releases/1.20.x/announcing-1.20.5/index.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+并修复了一些错误，提高了稳健性。
+
+本发布说明描述了 Istio 1.20.4 和 Istio 1.20.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同情况的重复主机将导致路由被 Envoy 拒绝的错误。
+  ([Issue #49368](https://github.com/istio/istio/issues/49368))
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:20]: Wait this [#49368](https://github.com/istio/istio/issues/49368) is not a issue, but an automator job...

File: content/zh/news/releases/1.20.x/announcing-1.20.5/index.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+本次发布实现了 4 月 8 日公布的安全更新 [`ISTIO-SECURITY-2024-002`](/zh/news/security/istio-security-2024-002)
+并修复了一些错误，提高了稳健性。
+
+本发布说明描述了 Istio 1.20.4 和 Istio 1.20.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同情况的重复主机将导致路由被 Envoy 拒绝的错误。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:19]:
```suggestion
- **修复** 修复了当 `VirtualService` 包含不同大小写的重复主机将导致路由被 Envoy 拒绝的错误。
```

File: content/zh/news/releases/1.21.x/announcing-1.21.1/index.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+本次发布实现了 4 月 8 日公布的安全更新 [`ISTIO-SECURITY-2024-002`](/zh/news/security/istio-security-2024-002)
+并修复了一些错误，提高了稳健性。
+
+本发布说明描述了 Istio 1.21.0 和 Istio 1.21.1 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同情况的重复主机将导致路由被 Envoy 拒绝的错误。
```
Comment (hanxiaop) [inline content/zh/news/releases/1.21.x/announcing-1.21.1/index.md:19]: same
```suggestion
- **修复** 修复了当 `VirtualService` 包含不同大小写的重复主机将导致路由被 Envoy 拒绝的错误。
```

File: content/zh/news/releases/1.20.x/announcing-1.20.5/index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:20]: Let me check it.

File: content/zh/news/releases/1.20.x/announcing-1.20.5/index.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+本发布说明描述了 Istio 1.20.4 和 Istio 1.20.5 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同大小写的重复主机将导致路由被 Envoy 拒绝的错误。
+  ([Issue #49368](https://github.com/istio/istio/issues/49368))
+
+- **修复** 修复了由于 ECDS 配置的存在，依赖于 Envoy 配置转储的命令无法工作的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:22]:
```suggestion
- **修复** 修复了由于存在 ECDS 配置使得依赖 Envoy 配置转储的命令无法工作的问题。
```

File: content/zh/news/releases/1.21.x/announcing-1.21.1/index.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+本发布说明描述了 Istio 1.21.0 和 Istio 1.21.1 之间的不同之处。
+
+{{< relnote >}}
+
+## 变更 {#changes}
+
+- **修复** 修复了当 `VirtualService` 包含不同大小写的重复主机将导致路由被 Envoy 拒绝的错误。
+  ([Issue #49368](https://github.com/istio/istio/issues/49368))
+
+- **修复** 修复了由于 ECDS 配置的存在，依赖于 Envoy 配置转储的命令无法工作的问题。
```
Comment (windsonsea) [inline content/zh/news/releases/1.21.x/announcing-1.21.1/index.md:22]:
```suggestion
- **修复** 修复了由于存在 ECDS 配置使得依赖 Envoy 配置转储的命令无法工作的问题。
```

**Review by windsonsea — COMMENTED** [review]
Nits

File: content/zh/news/releases/1.20.x/announcing-1.20.5/index.md (hunk 1 above)
Comment (hanxiaop) [reply content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:20]: Yes, looks like the original English content is incorrect as well.

Comment (wilsonwu) [reply content/zh/news/releases/1.20.x/announcing-1.20.5/index.md:20]: Let me correct.

Comment (wilsonwu) [reply content/zh/news/releases/1.20.x/announcing-1.20.5/index.md]: Done in Chinese part.

**Review by hanxiaop — APPROVED** [review]

**Review by zirain — APPROVED** [review]

**Pull request merged**